assemblers/org.osbuild.ostree.commit

```diff
@@ -15,9 +15,13 @@ Takes a file system tree that is already conforming to the ostree
 system layout[1] and commits it to an archive repository.
 
 The repository is located at the `/repo` directory and additional
-metadata is stored in `/commit.id` and `/compose.json` which contain
-the commit id and the compose information respectively.
+metadata is stored in `/compose.json` which contain the commit
+compose information respectively.
```

Comment:
The `respectively` is kinda redundant with only one file mentioned, isn't it?

Reply:
Yeah, leftover when there were two. Same with the `contain` above.